seqr/utils/elasticsearch/constants.py

```diff
@@ -109,25 +107,34 @@
         'filter_AF': ['gnomad_genomes_AF_POPMAX_OR_GLOBAL'],
     },
     'gnomad_svs': {},
-}
-
-MITO_POPULATION = {
+    'callset_heteroplasmy': {
+        'AN': 'AN',
+        'AC': 'AC_het',
+        'AF': 'AF_het',
+    },
     'gnomad_mito': {},
+    'gnomad_mito_heteroplasmy': {
+        'AN': 'gnomad_mito_AN',
+        'AC': 'gnomad_mito_AC_het',
+        'AF': 'gnomad_mito_AF_het',
+    },
     'helix': {},
+    'helix_heteroplasmy': {
+        'AN': 'helix_AN',
+        'AC': 'helix_AC_het',
+        'AF': 'helix_AF_het',
+    }
 }
-POPULATIONS.update(MITO_POPULATION)
 
 POPULATION_FIELD_CONFIGS = {
-    'AF': {'format_value': float},
+    'AF': {'format_value': lambda val: float(val) if val is not None else None, 'default_value': None},
```

Comment:
This will completely break every single population in seqr. If a field exists in the index but is missing from a particular variant, that means that variant has an AF of 0. This change can not be integrated. Can you please explain why this change was necessary for mito so I can figure out how to make it work better?

Reply:
I was hesitating to make this change, but forgot to ask. The are two reasons that we need to keep None, one is to avoid the null value gnomad_mito_af/gnomad_mito_af_het/helix_af/helix_af_het to become 0, the other is that we use af_het to detect if it is a mitochondrial calltype. If don't add `'default_value': None` then we can't use the af_het field to detect wether it is a mito call type because all the null value af_hets become 0.
Is there any way to fix the other populations' issue caused by the change?

Reply:
But for mito variants, doesn't a null  gnomad_mito_af/gnomad_mito_af_het/helix_af/helix_af_het MEAN that the value is 0? We've joined with our reference data so if the variant is missing in that reference data set it means its not there aka has an AF of 0

Reply:
I thought that when a variant is missing in a reference data set, it means the data set doesn't have the info about the variant. Does it also mean the reference population doesn't have the variant?

Reply:
If a reference data set does not have a variant, that means the variant is not in the reference data set. If the variant is not in a population frequency data set, that means the frequency is 0. If the frequency was anything other than 0, it would be in the datatset